modules/scripts-utils/dtbook-utils/src/test/xprocspec/test_fix.xprocspec

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<x:description xmlns:x="http://www.daisy.org/ns/xprocspec"
+               xmlns:px="http://www.daisy.org/ns/pipeline/xproc"
+               script="http://www.daisy.org/pipeline/modules/dtbook-utils/library.xpl">
+
+    <x:scenario label="Tidy, repair and narrator execution test">
+        <x:call step="px:dtbook-fix">
+            <x:input port="source">
+                <x:document type="inline">
+                    <dtbook xmlns="http://www.daisy.org/z3986/2005/dtbook/" version="2005-3">
+                        <head>
+                            <meta name="dc:Date" content="2023-06-08+02:00"/>
+                            <title>Title</title>
+                            <meta name="dc:Title" content="Title"/>
+                        </head>
+                        <book>
+                            <bodymatter>
+                                <level1>
+                                    <p>Blah</p>
+                                    <p/>
+                                </level1>
+                            </bodymatter>
+                        </book>
+                    </dtbook>
+                </x:document>
+            </x:input>
+            <x:option name="tidy" select="true()" />
+            <x:option name="repair" select="true()" />
+            <x:option name="narrator" select="true()" />
+            <x:option name="documentLanguage" select="'fr-FR'"/>
+            <x:option name="publisher" select="'DAISY Consortium'"/>
+        </x:call>
+        <x:context label="the processed dtbook 2005-3">
+            <x:document type="port" port="result"/>
+        </x:context>
+        <x:expect label="the dtbook 2005-3 should be as expected"
+                  type="custom" href="http://www.daisy.org/xprocspec/custom-assertion-steps.xpl" step="x:compare-except-ids">
+            <x:document type="inline">
+                <dtbook xmlns="http://www.daisy.org/z3986/2005/dtbook/" version="2005-3" xml:lang="fr-FR">
+                    <head>
+                        <meta name="dc:Publisher" content="DAISY Consortium"/>
+                        <meta name="dc:Language" content="fr-FR"/>
+                        <meta name="dtb:uid" content="AUTO-UID-d990e3"/>
```

Comment:
This is brittle. We can perhaps change it later with `http://www.daisy.org/pipeline/modules/common-utils/generate-id.xsl`.